perun-ldapc-ada/src/main/resources/perun-ldapc.xml

```diff
@@ -337,7 +337,7 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 							<value>cz.metacentrum.perun.core.api.Group</value>
 						</list>
 					</property>
-					<property name="pattern" value="Group:\[(.|\s)*\] was moved" />
+					<property name="pattern" value="Group:\[.*\] was moved$" />
```

Comment:
In old LDAPc we match it like this ```^Group:\\[(.*)\\] was moved\\.$``` (with ending dot, which is a part of auditer message). I'm not sure, that it will work here since you match end of string right after ``moved``.

Generally this is same for most of messages, but they are not consistent whether they have ending "dot" or not, so in most cases we don't match end of string and don't use ``Pattern.MULTILINE``, but ``Pattern.DOTALL``.